cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -658,3 +658,166 @@ Feature: JWT Authenticator - Fetch signing key
     """
     CONJ00035E Failed to decode token (3rdPartyError ='#<JWT::DecodeError: No key id (kid) found from token headers>')
     """
+
+  @sanity
+  Scenario: ONYX-15322: public-keys happy path
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+      - !variable public-keys
+      - !variable issuer
+      - !variable token-app-property
+
+      - !group hosts
+
+      - !permit
+        role: !group hosts
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project_id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I am the super-user
+    And I initialize remote JWKS endpoint with file "public-key-1" and alg "RS256"
+    And I successfully set authn-jwt public-keys variable to value from remote JWKS endpoint "public-key-1" and alg "RS256"
```

Comment:
add "" so it will be clear the variable that you are setting (same as you do for issuer and token-app-property)

Reply:
In this case "public-keys" is not the sentence variable. Only public-keys can accept this kind of value at the moment